docs/resources/cluster.md

```diff
@@ -1029,6 +1029,11 @@ Optional:
 - `disk_size` (String) Root disk size in gigabytes for the VM
 - `memory` (String) Memory associated with the node in megabytes
 
+
```

Comment:
nit: remove these extra lines, they are auto-generated every time, so just check the changes and revert these.